Week1/homework/app.js

```diff
@@ -1,11 +1,142 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
-
-  // Replace with your own code
-  console.log(bookTitles);
+const bookTitles = [
+  // Replace with your own book titles
+  'Nation_and_Novel',
+  'Robinson_Crusoe',
+  'Gullivers_Travels',
+  'Clarissa',
+  'Tom_Jones',
+  'Emma',
+  'Frankenstein',
+  'Nightmare_Abbey',
+  'Sybil',
+  'Jane_Eyre',
+];
+
+const myBooks = { 
+    Nation_and_Novel:{
+    name: 'Nation and Novel',
+    author: 'Patrick Parrinder',
+    language: 'English',
+  },
+  Robinson_Crusoe:{
+    name: 'Robinson Crusoe',
+    author: 'Daniel Defoe',
+    language: 'English',
+  },
+  Gullivers_Travels:{
+    name: 'Gulliver’s Travels',
+    author: 'Jonathan Swift',
+    language: 'English',
+  },
+  Clarissa:{
+    name: 'Clarissa',
+    author: 'Samuel Richardson',
+    language: 'English',
+  },
+  Tom_Jones:{
+    name: 'Tom Jones ',
+    author: 'Henry Fielding',
+    language: 'English',
+  },
+  Emma:{
+    name: 'Emma',
+    author: 'Jane Austen',
+    language: 'English',
+  },
+  Frankenstein:{
+    name: 'Frankenstein',
+    author: 'Mary Shelley',
+    language: 'English',
+  },
+  Nightmare_Abbey:{
+    name: ' Nightmare Abbey',
+    author: 'Thomas Love Peacock',
+    language: 'English',
+  },
+  Sybil:{
+    name: 'Sybil',
+    author: 'Benjamin Disraeli',
+    language: 'English',
+  },
+  Jane_Eyre :{
+    name: 'Jane Eyre',
+    author: 'Charlotte Brontë',
+    language: 'English',
+  },
+};
+const bookCover = {
+  Nation_and_Novel: 'https://i.postimg.cc/bwYx7JBp/L872-0-1.jpg',
+  Robinson_Crusoe: 'https://i.postimg.cc/vmJ6ZH31/IMG-3914.jpg',
+  Gullivers_Travels:'https://i.postimg.cc/0Q7tP0Lh/Gulliver-s-Travels.jpg',
+  Clarissa:'https://i.postimg.cc/cJysKgGN/f120c581f1c831d884d0ab1be66cb4ea.jpg',
+  Tom_Jones:'https://i.postimg.cc/43LtD7LV/1281362.jpg',
+  Emma:'https://i.postimg.cc/vTMQXXrz/Emma.jpg',
+  Frankenstein:'https://i.postimg.cc/brFBdwwt/Frankenstein.jpg',
+  Nightmare_Abbey:'https://i.postimg.cc/mDdsNRrb/Nightmare-Abbey.jpg',
+  Sybil:'https://i.postimg.cc/TP0MfGgr/Sybil.jpg',
+  Jane_Eyre:'https://i.postimg.cc/prY1YJCK/Jane-Eyre.jpg',
+};
+
+// Create a function to build a list of books including the required  information.
+function bookLibrary(createTheMainList) { 
```

Comment:
The name of the parameter `createTheMainList` suggests that it is a function. But it is actually an html element. Please rename the parameter to something more appropriate.